src/layers/basic.jl

```diff
@@ -56,24 +70,20 @@ end
 
 Calculate the forward results of each layers in Chain `c` with `input` as model input.
 """
-function activations(c::Chain, input)
-    extraChain(c.layers, input)
-end
+activations(c::Chain, input) = extraChain(Tuple(c.layers), input)
 
 function extraChain(fs::Tuple, x)
-    res = first(fs)(x)
-    return (res, extraChain(Base.tail(fs), res)...)
+  res = first(fs)(x)
+  return (res, extraChain(Base.tail(fs), res)...)
 end
-
 extraChain(::Tuple{}, x) = ()
 
 
-
 """
     Dense(in, out, σ=identity; bias=true, init=glorot_uniform)
     Dense(W::AbstractMatrix, [bias, σ])
 
-Create a traditional `Dense` layer, whose forward pass is given by:
+Create a traditional fully-connected layer, whose forward pass is given by:
```

Comment:
I'd leave this be for now
```suggestion
Create a traditional `Dense` layer, whose forward pass is given by:
```

Reply:
Sure, this is orthogonal